rustls/tests/common/mod.rs

```diff
@@ -1040,3 +1040,18 @@ impl RawTls {
         f(msg);
     }
 }
+
+pub fn with_confidentiality_limit(
+    original: SupportedCipherSuite,
+    limit: u64,
+) -> rustls::Tls13CipherSuite {
```

Comment:
I don't feel strongly, but since there's only one call-site for this fn it feels like it might be less cognitive overhead to inline this in `test_automatic_refresh_traffic_keys()`, or otherwise put it alongside that test instead of in `common/mod.rs` (at least until there's more callers?)

Reply:
Makes sense :+1: